modules/installation-aws-permissions-iam-roles.adoc

```diff
@@ -3,16 +3,23 @@
 // * installing/installing_aws/installing-aws-account.adoc
 
 [id="installation-aws-permissions-iam-roles_{context}"]
-= Required AWS permissions for IAM roles
+= IAM Policies and AWS Authentication
 
-You have the option of defining your own IAM roles that are applied to the instance profiles of your machines created by the installation program. You can specify existing IAM roles by defining the `controlPlane.platform.aws.iamRole` and `compute.platform.aws.iamRoleThis` fields in the `install-config.yaml` file. You can use these fields to match naming schemes and include predefined permissions boundaries for your IAM roles.
+This section presents how authentication works for AWS resources and how to set IAM policies that grants the minimal required privileges for the cluster instances. However, be aware that the stricly minimal set of permissions is dependent on each cluster configuration and operations. You can start with the policies below, and adapt if needed, or use the https://docs.aws.amazon.com/IAM/latest/UserGuide/what-is-access-analyzer.html[IAM Access Analyzer] to generate the IAM Policies based on CloudTrail logs. 
+
+= Required permissions for IAM Instance Profiles
+
+By default, the installer creates IAM instance profiles for the bootstrap, control plane and worker instances with the necessary permissions. The control plane and bootstrap instances use this instance profile to make AWS API calls.
```

Comment:
> The control plane and bootstrap instances use this instance profile to make AWS API calls.

There are 3 instance profiles will be created for Bootstrap, Control Plane and Compute node, what's the `this instance profile`, we need to make it clear.